### ISSUE TITLE
Question about `neuritefunc` features and Population.

### ISSUE THREAD

**Issue by asanin-epfl** [BlueBrain/NeuroM#943]
@eleftherioszisis @arnaudon please clarify whether it is intentional or accidental.

Currently if a feature from [neuritefunc](https://github.com/BlueBrain/NeuroM/blob/master/neurom/features/neuritefunc.py) is called with a neuron population as its argument then it will be applied to the whole population as a single neuron. In order to get this feature per neuron, the feature has to be recreated under different name. For example lets consider the feature [n_neurites](https://github.com/BlueBrain/NeuroM/blob/978d39fc4562027a72fc515ee76ee5195e0b12b5/neurom/features/neuritefunc.py#L83) and its recreated version [number_of_neurites](https://github.com/BlueBrain/NeuroM/blob/978d39fc4562027a72fc515ee76ee5195e0b12b5/neurom/features/neuritefunc.py#L205).

```python
from neurom import load_neurons
from neurom.features import get

pop = load_neurons('path/to/folder/with/3/morphologies')
n = get('n_neurites', pop)
# n will be a list with a single element, the total number of neurites in `pop`
n = get('number_of_neurites', pop)
# n will contain 3 elements, number of neurites per neuron.
```

My question, are you aware of this? Are you aware that if a feature does not have a recreated auxiliary, it will produce results for a neuron population as a huge single neuron? There are plenty of features in `neuritefunc` that don't have recreated auxiliary functions for them.

**Comment by arnaudon** [comment]
This does not sound so nice indeed... and these 2 names of features even worse... do you think it would be possible to have an option 'total' or 'per_neuron' or something like that? tbh, I don't really know... maybe @lidakanari knows more, as she was part of neurom dev since its early days.

**Comment by asanin-epfl** [comment]
solved in 3.0.0
